eda_analysis/eda_analysis.py

```diff
@@ -299,6 +308,9 @@ def describe_num_var(dataframe, num_vars):
     ).facet(
         facet='variable:N',
         columns=3
+    ).resolve_scale(
+        x='independent',
+        y='independent'
```

Comment:
It seems to work for only one axis at a time. For this set up, since we are faceting on columns, it will work for y and not x. 

I found a suggestion from Jake on a similar issue:

    plot = alt.vconcat(
      *(alt.Chart()
          .transform_filter(alt.datum.variable == var)
          .mark_bar()
          .encode(
            alt.X("value",bin=alt.Bin(maxbins=30)),
            alt.Y("count()")
          )
        for var in sorted(df_to_plot.variable.unique())
          ),
      data=df_to_plot
        )

Seems to create a histogram with flexible x and y axis values, but only one per row. Code suggestion from: https://github.com/altair-viz/altair/issues/2015

Reply:
It seems to work, thank you, Sarah.

Reply:
However, now we can only have the plots in one column. But I guess that's the best we can do with Altair.